litellm/proxy/proxy_server.py

```diff
@@ -1073,6 +1073,8 @@ async def user_api_key_auth(
             if not _is_user_proxy_admin(user_id_information):  # if non-admin
                 if route in LiteLLMRoutes.openai_routes.value:
                     pass
+                elif request['route'].name in LiteLLMRoutes.openai_route_names.value:
```

Comment:
can you add a test for this? 

this might be a good place for it - https://github.com/BerriAI/litellm/blob/d208dedb35b2cec4ade6a2acb63f46eb84b49b3c/litellm/tests/test_key_generate_prisma.py#L4

Reply:
I think I'm confused about how this relates to prisma and key generation, maybe because I haven't had enough coffee yet.

Can you give me an example or outline to bootstrap me?

Reply:
- create a virtual key (non-admin) 
- call user_api_key_auth with it, with the route you made this PR for 
- check if user_api_key_auth let's the request go through 

e.g. - https://github.com/BerriAI/litellm/blob/d208dedb35b2cec4ade6a2acb63f46eb84b49b3c/litellm/tests/test_key_generate_prisma.py#L149

Reply:
How does d5b2e8e7e look?